lib/pages/login/login_page.dart

```diff
@@ -0,0 +1,245 @@
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:image_picker/image_picker.dart';
+import 'package:intl_phone_field/intl_phone_field.dart';
+
+class LoginPage extends StatefulWidget {
+  static const String routeName = '/loginpage';
+
+  const LoginPage({super.key});
+
+  @override
+  State<LoginPage> createState() => _LoginPageState();
+}
+
+class _LoginPageState extends State<LoginPage> {
+  //form key
+  final _formKey = GlobalKey<FormState>();
+
+  //image variables
+  File? _image;
+  ImagePicker picker = ImagePicker();
+
+  //function to click photo from camera
+  _imgFromCamera() async {
+    XFile? image =
+        await picker.pickImage(source: ImageSource.camera, imageQuality: 50);
+
+    setState(() {
+      _image = File(image!.path);
```

Comment:
What if the user presses the back button and no image is taken?
image! will throw error

Handle that check that if image is not null then only update